### PR TITLE
added dockerfile to create a docker instnace runnning the latest open…

### PR DESCRIPTION
**Pull request by zachrussell12**
…jdk and jre. added a batch file to orchestrate everything with on click. added temporary instructions to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,22 @@ Zachary Russell             | Writer and Coder
...
+
+Double-click on the 'LaunchRoundTablerInstance.bat' to automatically build and start the Docker container.
+
+Open Docker Desktop and open the CLI on the instance named 'roundtabler'
+
+Navigate to the src directory using:
+
+	`cd src`
+
+Look at the list of parameters to get started:
```
Comment (ScottPiersall) [inline README.md]: "To review the list of options please see:"

Comment (zachrussell12) [reply README.md]: made those changes. i just threw those there for other people on our team. we can make the read me more official later.

Comment (ScottPiersall) [reply README.md]: Ok. understood.  docker looks great! nice work!

**Review by ScottPiersall — APPROVED** [review]
Nice work Zach!!!!

**Pull request merged**